etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/localfile/FilePartnerMetadataStorage.java

```diff
@@ -65,14 +65,18 @@ public Optional<PartnerMetadata> readMetadata(final String uniqueId)
 
     @Override
     public void saveMetadata(final PartnerMetadata metadata) throws PartnerMetadataException {
-        Path metadataFilePath = getFilePath(metadata.uniqueId());
+        Path metadataFilePath = getFilePath(metadata.receivedSubmissionId());
         try {
             String content = formatter.convertToJsonString(metadata);
             Files.writeString(metadataFilePath, content);
-            logger.logInfo("Saved metadata for " + metadata.uniqueId() + " to " + metadataFilePath);
+            logger.logInfo(
+                    "Saved metadata for "
```

Comment:
I think you want to parameterize the inputs for log statements.  So something like 

```suggestion
                    logger.logInfo("Saved metadata for {} to {}", metadata.receivedSubmissionId(), metadataFilePath);
```

It's cleaner and more efficient

Reply:
Good catch, I'll do that